src/Makefile.am

```diff
@@ -19,6 +21,7 @@ noinst_HEADERS = \
 	signature.h \
 	slot.h \
 	store.h \
+	store_internals.h \
```

Comment:
We already have a store.h, no need for a new header

Reply:
I moved the p11prov_store_direct_fetch declaration to store.h